roadmap/implementers-guide/src/node/subsystems-and-jobs.md

```diff
@@ -11,3 +11,427 @@ Since this goal of determining when to start and conclude work relative to a spe
+    participant CB as CandidateBacking
+    participant CV as CandidateValidation
+    participant PV as Provisioner
+    participant SD as StatementDistribution
+    participant PD as PoVDistribution
+
+    CS ->> CB: Second
+    % fn validate_and_make_available
+    CB -->> CV: ValidateFromChainState
+
+    Note over CB,CV: There's some complication in the source, as<br/>candidates are actually validated in a separate task.
+
+    alt valid
+        Note over CB: This is where we transform the CandidateReceipt into a CommittedCandidateReceipt
+        % CandidateBackingJob::sign_import_and_distribute_statement
+        % CandidateBackingJob::import_statement
+        CB ->> PV: ProvisionableData::BackedCandidate
+        % CandidateBackingJob::issue_new_misbehaviors
+        opt if there is misbehavior to report
+            CB ->> PV: ProvisionableData::MisbehaviorReport
+        end
+        % CandidateBackingJob::distribute_signed_statement
+        CB ->> SD: Share
+        % CandidateBackingJob::distribute_pov
+        CB ->> PD: DistributePoV
+    else invalid
+        CB ->> CS: Invalid
+    end
+```
+
+At this point, you'll see that control flows in two directions: to `StatementDistribution` to distribute
+the `SignedStatement`, and to `PoVDistribution` to distribute the `PoV`. However, that's largely a mirage:
+while the initial implementation distributes `PoV`s by gossip, that's inefficient, and will be replaced
+with a system which fetches `PoV`s only when actually necessary.
+
+> TODO: figure out more precisely the current status and plans; write them up
+
+Therefore, we'll follow the `SignedStatement`. The `StatementDistribution` subsystem is largely concerned
+with implementing a gossip protocol:
+
+```mermaid
+sequenceDiagram
+    participant SD as StatementDistribution
+    participant NB as NetworkBridge
+    participant Listener
+
+    alt On receipt of a<br/>SignedStatement from CandidateBacking
+        % fn circulate_statement_and_dependents
+        SD ->> NB: SendValidationMessage
+
+        Note right of NB: Bridge sends validation message to all appropriate peers
+    else On initialization, from other subsystems:
+        Listener ->> SD: RegisterStatementListener
+    else On receipt of peer validation message
+        NB ->> SD: NetworkBridgeUpdateV1
+
+        % fn handle_incoming_message
+        alt if we aren't already aware of the relay parent for this statement
+            SD ->> NB: ReportPeer
+        else the statement corresponds to our View
+            Note over SD,Listener: Forward the statement to each registered listener
+            SD ->> Listener: SignedFullStatement
+        end
+
+        % fn circulate_statement
+        opt if we know of peers who haven't seen this message, gossip it
+            SD ->> NB: SendValidationMessage
+        end
+    end
+```
+
+But who are these `Listener`s who've asked to be notified about incoming `SignedStatement`s?
+Nobody, as yet.
```

Comment:
A possible candidate that consume this information could be the `Disputes`  module, but that's far from set in stone.